packages/js/product-editor/src/components/custom-fields/custom-fields.tsx

```diff
@@ -163,14 +163,16 @@ export function CustomFields( {
 
 			{ showCreateModal && (
 				<CreateModal
+					values={ customFields }
 					onCreate={ handleCreateModalCreate }
 					onCancel={ handleCreateModalCancel }
 				/>
 			) }
 
-			{ selectedCustomField && (
+			{ selectedCustomFieldIndex != undefined && (
```

Comment:
I think that this should be `!==` instead of `!=`

Reply:
Thanks @octaedro done -> [01f489e](https://github.com/woocommerce/woocommerce/pull/46703/commits/01f489e92e50177258a2355b645ee24d2b0e61ae)